health-services/geopode-adapter/src/main/resources/start.sh

```
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if [[ -z "${JAVA_OPTS}" ]];then
+export JAVA_OPTS="-Xmx64m -Xms64m"
+fi
+
+if [ x"${JAVA_ENABLE_DEBUG}" != x ] && [ "${JAVA_ENABLE_DEBUG}" != "false" ]; then
+java_debug_args="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=${JAVA_DEBUG_PORT:-5005}"
+fi
+
+exec java ${java_debug_args} ${JAVA_OPTS} ${JAVA_ARGS}  -jar /opt/egov/*.jar
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)